### ISSUE TITLE
mention similar projects in readme

### ISSUE THREAD

**Issue by milahu** [evnbr/bindery#132]
imho its good sport to mention similar projects

i only know [pagedjs](https://github.com/pagedjs/pagedjs) which is more active than bindery

**Comment by evnbr** [comment]
Yes, pagedjs seems great and appears to be much more mature than bindery, lots of great customization options. I haven't had the chance to revisit this project recently but agree on linking out.

In addition to Paged which is also open-source, there's also the even more professional-quality https://www.princexml.com/ , which I believe is licensed. 

Both of those lean more into CSS based customization.
